FORKED_CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+<!--
+Guiding Principles:
+
+Changelogs are for humans, not machines.
+There should be an entry for every single version.
+The same types of changes should be grouped.
+Versions and sections should be linkable.
+The latest version comes first.
+The release date of each version is displayed.
+Mention whether you follow Semantic Versioning.
+
+Usage:
+
+Change log entries are to be added to the Unreleased section under the
+appropriate stanza (see below). Each entry should ideally include a tag and
+the Github issue reference in the following format:
+
+* (<tag>) \#<issue-number> message
+
+The issue numbers will later be link-ified during the release process so you do
+not have to worry about including a link manually, but you can if you wish.
+
+Types of changes (Stanzas):
+
+"Features" for new features.
+"Improvements" for changes in existing functionality.
+"Deprecated" for soon-to-be removed features.
+"Bug Fixes" for any bug fixes.
+"Client Breaking" for breaking Protobuf, gRPC and REST routes used by end-users.
+"CLI Breaking" for breaking CLI commands.
+"API Breaking" for breaking exported APIs used by developers building on SDK.
+"State Machine Breaking" for any changes that result in a different AppState given same genesisState and txList.
+Ref: https://keepachangelog.com/en/1.0.0/
+-->
+
+# Changelog
+
+
+## Current - 2022-10-21
+
+### Bug Fixes
+
+* [\#2](https://github.com/sei-protocol/sei-cosmos/pull/2) Fix GRPC bug
+
+### Improvements
+
+* [\#1](https://github.com/sei-protocol/sei-cosmos/pull/1) Integrate Cosmos with sei-tendermint and ABCI++
+* [\#14](https://github.com/sei-protocol/sei-cosmos/pull/14) Integrate Cosmos with Tendermint tracing
+* (x/auth/vesting) [\#11652](https://github.com/cosmos/cosmos-sdk/pull/11652) Add util functions for `Period(s)`
+
+
+### Features
+* [\#17](https://github.com/sei-protocol/sei-cosmos/pull/17) Support SR25519 algorithm for client transaction signing
+* [\#23](https://github.com/sei-protocol/sei-cosmos/pull/23) Add priority to CheckTx based on gas fees
+* (x/accesscontrol)[\#24](https://github.com/sei-protocol/sei-cosmos/pull/24) Add AccessControl module
```

Comment:
We should also include the change that adds the dependency dag to access control, right? It initially was put in sei chain and then refactored to sei-cosmos because it fits better in the access control module

Reply:
@udpatil can you point me to the PR?

Reply:
https://github.com/sei-protocol/sei-cosmos/pull/36